addons/packages/pinniped/0.12.0/package.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  name: pinniped.community.tanzu.vmware.com.0.12.0
+  namespace: pinniped
+spec:
+  refName: pinniped.community.tanzu.vmware.com
+  version: 0.12.0
+  releaseNotes: "pinniped 0.12.0 https://github.com/vmware-tanzu/pinniped/releases/tag/v0.12.0"
+  licenses:
+    - "Apache 2.0"
+  template:
+    spec:
+      fetch:
+        - imgpkgBundle:
+            image: projects.registry.vmware.com/tce/pinniped:v0.12.0 #! TODO: this doesn't exist :) what should we use here?
```

Comment:
what image should we put here?

honestly i kinda feel silly about this because i have no idea how this in-tree package is used by OSS users...

Reply:
You push the package to an OCI Registry and get the URL/SHA256-Digest. Use that value here.